SConstruct

```diff
@@ -1,4 +1,74 @@
 # -*- python -*-
-from lsst.sconsUtils import scripts
-# Python-only package
-scripts.BasicSConstruct("ci_ap", disableCc=True, noCfgFile=True)
+import os
+from SCons.Script import Environment, GetOption, Default
+from lsst.sconsUtils.utils import libraryLoaderEnvironment
+
+env = Environment(ENV=os.environ)
+env["ENV"]["OMP_NUM_THREADS"] = "1"  # Disable threading
+
+
+def getExecutableCmd(package, script, *args, directory=None):
+    """
+    Given the name of a package and a script or other executable which lies
+    within the given subdirectory (defaults to "bin"), return an appropriate
+    string which can be used to set up an appropriate environment and execute
+    the command.
+    This includes:
+    * Specifying an explict list of paths to be searched by the dynamic linker;
+    * Specifying a Python executable to be run (we assume the one on the
+      default ${PATH} is appropriate);
+    * Specifying the complete path to the script.
+    """
+    if directory is None:
+        directory = "bin"
+    cmds = [libraryLoaderEnvironment(), "python",
+            os.path.join(env.ProductDir(package), directory, script)]
+    cmds.extend(args)
+    return " ".join(cmds)
```

Comment:
I guess this is required for `env.Command()`, but you should note that anything in `*args` or `directory` containing spaces needs to be appropriately quoted for the shell.

Reply:
I stole this function from ci_hsc, which doesn't do any such quoting. Should I try to sort that out here, or can we be pretty sure that we'll never trigger that problem in real usage?

Reply:
I think a docstring warning is good enough.  macOS is the place I worry about the most.